client/src/componets/routes/CreateRecipe.js

```diff
@@ -8,11 +8,11 @@ function RecipeCreate() {
   const navigate = useNavigate();
   const [item, setItem] = useState({
     title: '',
-    Rating: '',
+    ingredients: '',
```

Comment:
Ingredients is an array of strings, so it should contain a value of `[""]` as such. Directions is also an array of strings so, `[""] ` is its value as well